back/babble/src/test/java/gg/babble/babble/service/RoomServiceTest.java

```diff
@@ -39,6 +48,23 @@
     @Autowired
     private TagService tagService;
 
+    private CreatedRoomResponse savedRoom;
+
+    @BeforeEach
+    void setUp() {
+        savedRoom = roomService.create(
+            new RoomRequest(1L,
+                Arrays.asList(
+                    new TagRequest(tagService.findByName(실버).get(0).getId()),
```

Comment:
제 취향을 아시겠지만 Optional.orElseThrow() 처리 해주는 것을 선호합니다!

```suggestion
                    new TagRequest(tagService.findByName(실버).orElseThrow(BabbleNotFoundException::new).getId()),
```

Reply:
return 값이 List라서 `orElseThrow`가 안되는 것으로 알고 있습니다. 또한 추후 Id 기반 테스트를 전부 고쳐야해서 이 부분도 추후 리팩터링 사항에 해당할 것 같습니다.

Reply:
제가 잘못봤습니다.
Optional<List<Tag>> 가 반환된다고 생각했네요. 😭